clang/test/CodeGenSYCL/Inputs/debug-info-checksum.cpp

```diff
@@ -0,0 +1,14 @@
+#include "Inputs/sycl.hpp"
+
+template <typename name, typename Func>
+__attribute__((sycl_kernel)) void kernel(const Func &kernelFunc) {
```

Comment:
How about using some kernel from `Inputs/sycl.hpp?`

Reply:
Thanks, simplified the test in a54b24a9bdcac9a977afc73d00540d0657c4255f